packages/stream_video_flutter/lib/src/call_screen/lobby_view.dart

```diff
@@ -91,20 +95,17 @@ class _StreamLobbyViewState extends State<StreamLobbyView> {
 
   void onJoinCallPressed() {
     _isJoiningCall = true;
```

Comment:
Since we are not providing those tracks anymore I think we don't need `_isJoiningCall` flag - we should close both tracks on dispose always.

Reply:
Let's fix it in separate PR